test/binCases/errors/clean-webpack-options/stdin.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
```

Comment:
👍 I understand that testing it isolated doesn't make much sense, but having some test checking for correct error output is already better then nothing :)